kubeportal/admin_views.py

```diff
@@ -0,0 +1,43 @@
+from django.views.generic.base import TemplateView
+from django.contrib.auth import get_user_model
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.shortcuts import redirect
+from django.conf import settings
+from django.contrib import admin
+from kubeportal import kubernetes, models
+from kubeportal.models import KubernetesNamespace, KubernetesServiceAccount
+from datetime import datetime, timedelta
+
+def sync_view(request):
+    kubernetes.sync(request)
+    return redirect('admin:index')
+
+class CleanupView(LoginRequiredMixin, TemplateView):
+    template_name = "admin/backend_cleanup.html"
+
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+        context['site_header'] = settings.BRANDING + " (Admin Backend)"
+        context['title'] = "Clean Up"
+
+        User = get_user_model()
+
+        visible_namespaces = KubernetesNamespace.objects.filter(visible=True)
+        context['namespaces_no_portal_users'] = [ns for ns in visible_namespaces if ns.service_accounts.count() == 1]
```

Comment:
This produces A LOT of database queries. Pleasde try to make it a Django filter.